src/library/scala/collection/immutable/List.scala

```diff
@@ -97,7 +97,7 @@ sealed abstract class List[+A] extends AbstractSeq[A]
 
   // New methods in List
 
-  /** Adds an element at the beginning of this list.
+  /** A copy of this list with an element prepended.
```

Comment:
This wording suggests to me that all elements are copied and the operation would be `O(N)`. It doesn't seem like an improvement to me. Can you explain what you found unclear beforehand?